ci-operator/config/openshift/loki/openshift-loki-release-5.7.yaml

```diff
@@ -72,7 +123,7 @@ tests:
       OO_PACKAGE: loki-operator
```

Comment:
Can not comment on the correct line:

Should `OO_CHANNEL` be `stable-5.7`?

Reply:
There is no `stable-5.7` in the annotations yet.

Reply:
There should be. It's just that it exists on both `stable` and `stable-5.7`. But I guess you're right. This is stable because 5.8 isn't out yet. :+1: